src/index.html

```diff
@@ -6,8 +6,44 @@
           content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
     <title>Product cards</title>
     <link rel="stylesheet" href="./style.css">
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com">
+    <link href="https://fonts.googleapis.com/css2?family=Roboto:wght@400;500;700&display=swap" rel="stylesheet">
   </head>
   <body>
-    <h1>Product cards</h1>
+    <div class="card" data-qa="card">
+      <div class="card-image">
+        <img src="./images/imac.jpeg" alt="imac">
+      </div>
+      <div class="card-name">
+          APPLE A1419 iMac 27" Retina 5K Monoblock (MNED2UA/A)
+      </div>
+      <div class="card-code">
+          Product code: 195434
+      </div>
```

Comment:
1. use semantic tags for text elements. Apply changes for similar tags as well
2. use BEM blocks
```suggestion
      <h2 class="card__title">
          APPLE A1419 iMac 27" Retina 5K Monoblock (MNED2UA/A)
      </h2>
      <p class="card__code">
          Product code: 195434
      </p>
```